tests/testthat/test-CSCI.R

```diff
@@ -0,0 +1,4 @@
+test_that("CSCI runs on bugs_stations data and produces previously produced headers", {
+  results <- suppressWarnings(CSCI(bugs = bugs_stations[[1]], stations = bugs_stations[[2]]))
```

Comment:
I added suppressWarnings because the function produces a lot of warnings (I counted 13), and it seemed like scope creep to chase those down in this Pull Request.